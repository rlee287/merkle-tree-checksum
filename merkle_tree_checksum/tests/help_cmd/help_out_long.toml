bin.name = "merkle_tree_checksum"
args = "--help"
status.code = 1
