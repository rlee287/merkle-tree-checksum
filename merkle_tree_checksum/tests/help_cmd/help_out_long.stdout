Computes a Merkle Tree hash over the given files

Usage: merkle_tree_checksum [OPTIONS] <COMMAND>

Commands:
  generate-hash  Generates Merkle tree hashes
  verify-hash    Verify Merkle tree hashes
  help           Print this message or the help of the given subcommand(s)

Options:
  -q, --quiet...
          Specify once to hide progress bars. Specify twice to suppress all output besides errors.

  -j, --jobs <jobs>
          Specify size of thread pool for hashing. It is recommended to leave at least one CPU free
          for the main thread to read/write hashes. Adding more than 2 threads does not improve
          performance when I/O is the program bottleneck.
          
          [default: 4]

  -h, --help
          Print help (see a summary with '-h')

  -V, --version
          Print version

Supported hash functions are the SHA2 family, the SHA3 family, Blake2b/Blake2s, Blake3, and CRC32.
